luxonis_ml/data/augmentations/augmentations.py

```diff
@@ -85,41 +77,34 @@ def from_config(
             )
             if isinstance(curr_aug, A.ImageOnlyTransform):
                 pixel_augs.append(curr_aug)
-            elif isinstance(curr_aug, A.DualTransform):
-                spatial_augs.append(curr_aug)
             elif isinstance(curr_aug, BatchBasedTransform):
                 batch_size *= curr_aug.batch_size
                 batched_augs.append(curr_aug)
+            elif isinstance(curr_aug, A.DualTransform):
+                spatial_augs.append(curr_aug)
 
-        def _get_params(batch: bool = False):
-            suffix = "_batch" if batch else ""
+        def _get_params():
             return {
                 "bbox_params": A.BboxParams(
-                    format="coco",
-                    label_fields=[
-                        f"bboxes_classes{suffix}",
-                        f"bboxes_visibility{suffix}",
-                    ],
+                    format="albumentations",
```

Comment:
Any specific reason for switching to albumentations format from coco?

Reply:
Less pre- and post-processing